examples/source/2.guides/Using_the_Google_AMP_Cache/api.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const utils = require('@lib/utils');
+const {setMaxAge} = require('@lib/utils/cacheHelpers');
+const nunjucks = require('nunjucks');
```

Comment:
Can you please move this to a top-level example API. Thanks!

Reply:
Do you mean that it should be located within ```/examples/api```?

Reply:
genau

Reply:
PTAL